CHANGES.rst

```diff
@@ -1,6 +1,11 @@
 Changes
 =======
 
+TBR:
+------------------
+
+* --disable-ssl-check option to disable ssl verification.
```

Comment:
The feature looks good to me @ivanprado; the only thing is naming, as usual :) 

I know the parameter is named ssl in aiohttp, but SSL is deprecated a long time ago, and not used much by websites now (<4%); it's TLS all the way. Would it be possible to have another name for this option - tls, or something about certificates, or just something about being insecure?

Reply:
TIL TSL vs SSL. What do you think about https://github.com/zytedata/zyte-autoextract/pull/37/commits/015e5b1b57d330ac816ae0ab6ce21f484fd193b5?